src/ui/components/Events/ReplayInfo.tsx

```diff
@@ -51,12 +51,20 @@ function ReplayInfo({ setModal }: PropsFromRedux) {
           <MaterialIcon iconSize="xl">{icon}</MaterialIcon>
           <div>{summary}</div>
         </Row>
-        <Row>
-          <MaterialIcon iconSize="xl">language</MaterialIcon>
-          <div className="overflow-hidden overflow-ellipsis whitespace-pre" title={recording.url}>
-            {getDisplayedUrl(recording.url)}
-          </div>
-        </Row>
+
+        <div className="group">
+          <Row>
+            <MaterialIcon iconSize="xl" className="group-hover:text-primaryAccent">
+              language
+            </MaterialIcon>
+            <div className="overflow-hidden overflow-ellipsis whitespace-pre" title={recording.url}>
+              <a href={recording.url} target="_blank">
+                {getDisplayedUrl(recording.url)}
+              </a>
```

Comment:
Generally, it would be good to add `rel="noopener noreferrer"` to all `<a>` tags.

(related: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)

Reply:
On a non-blocking note, I could have sworn we had a <Link> component from a few months ago just so we didn't have to worry about this bit. Will make a point to check that out sometime today.

Reply:
Fixed!